data/fields/license_classes.json

```diff
@@ -0,0 +1,5 @@
+{
+    "key": "license_classes",
+    "type": "combo",
```

Comment:
I think the type `semiCombo` would be a better fit for this field, since typically more than one license class.

```suggestion
    "type": "semiCombo",
```